azurepipelines/e2e_test/README.md

```diff
@@ -156,33 +156,33 @@ The process for adapting or adding a new scenario can be accomplished by followi
-   1. Within this directory create the `setup.sh` script for installing the DeviceUpdate for IotHub agent on the VM as well as setup any and all needed dependencies (e.g. for an IotEdge device provisioned by the IoT Identity Service add the steps to install aziot-edge and configure it with the required information)
-   2. The next step is create the `devicesetup.py` script under the same directory that creates the device in the IotHub and the corresponding `du-config.json` file. This can create whatever provisioning information or work that you'd like to have on the device but you'll have to modify the `TerraformVMInitialization` step to use that information provided your scenario needs more than the standard du-config.json file.
-   3. For most cases the above two steps can be satisfied by just copying the `vm_setup` directory from any of the other scenarios and doing basic modifications. It should be easy to customize from there.
+    1. Within this directory create the `setup.sh` script for installing the DeviceUpdate for IotHub agent on the VM as well as setup any and all needed dependencies (e.g. for an IotEdge device provisioned by the IoT Identity Service add the steps to install aziot-edge and configure it with the required information)
+    2. The next step is create the `devicesetup.py` script under the same directory that creates the device in the IotHub and the corresponding `du-config.json` file. This can create whatever provisioning information or work that you'd like to have on the device but you'll have to modify the `TerraformVMInitialization` step to use that information provided your scenario needs more than the standard du-config.json file.
+    3. For most cases the above two steps can be satisfied by just copying the `vm_setup` directory from any of the other scenarios and doing basic modifications. It should be easy to customize from there.
 3. After creating the setup files for the VMs you need to add the scenario to the `TerraformVMInitialization` and `RunTestsAndPostResults` matrices. For both you need to modify the `matrix:` section to include the information for Terraform to create the VM an example of how Debian 10 might be added to the `TerraformVMInitialization` stage below. An explanation of what these are for and how they're used is provided in the sections above.
 
     ```yaml
-        debian-9-amd64:
-            distroName: debian-10
-            image_publisher: CentOS
-            image_offer: Debian10
-            image_sku: debian-10
-            image_version: latest
-            packagePattern: "*debian-90-amd64*/*.deb"
-            package_pipeline: "Azure.adu-private-preview.e2e-test"
-            scenarioSetupDir: './scenarios/debian-10-amd64/'
-            du_tarball_script: >-
-                tar -xvf /tmp/testsetup.tar.gz -C ./ &&
-                u=rwx,g=rwx,o=rx ./testsetup/setup.sh &&
-                ./testsetup/setup.sh
+    debian-9-amd64:
```

Comment:
Should be dabian-**10**-amd64?

And line 171 below too, debian-10... ?

Reply:
The Readme hasn't been managed well. I'm addressing that in a future PR (for x509 provisioning and we'll have to rewrite it with the rework).